contracts/Depository.sol

```diff
@@ -65,13 +65,16 @@ contract Depository is IErrorsTokenomics {
     event TreasuryUpdated(address indexed treasury);
     event BondCalculatorUpdated(address indexed _bondCalculator);
     event CreateBond(address indexed token, uint256 indexed productId, address indexed owner, uint256 bondId,
-        uint256 amountOLAS, uint256 tokenAmount, uint256 expiry);
-    event RedeemBond(uint256 indexed productId, address indexed owner, uint256 bondId);
-    event CreateProduct(address indexed token, uint256 indexed productId, uint256 supply, uint256 priceLP, uint256 expiry);
-    event CloseProduct(address indexed token, uint256 indexed productId);
+        uint256 amountOLAS, uint256 tokenAmount, uint256 maturity, uint256 startTime);
+    event RedeemBond(uint256 indexed productId, address indexed owner, uint256 bondId, uint256 redeemTime);
```

Comment:
Why do we need redeemTime here? that will presumably be the timestamp of the transaction which the log is part of?

Reply:
I am not yet seeing the benefit of the change to the events.

Reply:
Let's remove as it feels unnecessary